docs/usage/reading-list.md

```diff
@@ -0,0 +1,74 @@
+# Reading list
+
+Renovate's documentation has a lot of pages.
+To ease you into using Renovate we created reading lists.
+The reading lists contain the essential information for each type of user.
+
+How much you should read depends on how much you want to customize Renovate's behavior.
+
+## How to use this page
+
+We created reading lists for these types of users:
+
+- Beginners
+- Intermediate
+- Advanced
+
+Start with the "Beginners" reading list.
+If you're self-hosting or need to update private packages, complete the relevant reading lists for those.
+
+## I don't know where to start
+
+If you're new to Renovate, you should:
+
+- Use the Mend-hosted GitHub App, or let somebody else self-host Renovate for you
+- Stick with the `config:base` or `config:best-practices` preset
```

Comment:
```suggestion
- Stick with the `config:base` (or the upcoming `config:best-practices` preset)
```

We did not merge this PR yet, so I'll call it the "upcoming preset".

- #16100 

Or I can drop the reference for now, if you want.

Reply:
I would prefer to avoid "upcoming", so either we merge this after it's available, or drop it now